drivers/SmartThings/zigbee-button/src/zigbee-multi-button/somfy/init.lua

```diff
@@ -22,6 +22,7 @@ local function zdo_binding_table_handler(driver, device, zb_rx)
       return
     end
   end
+  driver:add_hub_to_zigbee_group(0x0000) -- fallback if no binding table entries found
```

Comment:
Looking at the DTH Somfy also needs to add the device to group 0x0000 in this case.